src/vs/platform/extensionManagement/common/extensionManagement.ts

```diff
@@ -105,7 +105,7 @@ export interface IExtensionManifest {
 	name: string;
 	publisher: string;
 	version: string;
-	engines: { vscode: string };
+	engines: { vscode: string; sqlops?: string };
```

Comment:
We should mark this and the changes in the file below as `// {{SQL CARBON EDIT}}`